Podfile.lock

```diff
@@ -65,4 +65,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 1b99113acef4f0ff99e478c800e9f465b013a083
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.10.0
```

Comment:
Unexpected commit, no?

Reply:
Updated my CocoaPods to the same version.